lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -384,41 +384,41 @@
         @include clearfix();
         @include padding(0, 10px, 0, 230px);
 
-         .course-item-info {
-          padding: 0;
+        .course-item-info {
+          position: relative;
           width: 100%;
+          padding: 0;
 
-          .university {
-            color: $lighter-base-font-color;
-            font-family: $sans-serif;
-            font-size: 16px;
-            font-weight: 400;
-            margin: 0 0 6px;
-            text-transform: none;
-            letter-spacing: 0;
+          .course-title {
+            margin-top: -($baseline/2);
+            a,
+            span {
+              display: block;
+              margin-bottom: ($baseline/3);
+              font-family: $sans-serif;
+              font-size: 34px;
+              line-height: 42px;
+              font-weight: 300;
+
+              &:hover,
```

Comment:
Is this needed to override some hover/focused states elsewhere? If so, can you add an inline note on why its needed?